app/main.py

```diff
@@ -1,3 +1,21 @@
 class OnlineCourse:
-    # write your code here
-    pass
+    def __init__(self, name: str,
+                 description: str,
+                 weeks: int | float) -> None:
+        self.name = name
+        self.description = description
+        self.weeks = weeks
+
+    def __str__(self) -> str:
+        return self.description
+
+    @staticmethod
+    def days_to_weeks(days: int) -> int:
+        result = days / 7
+        return result if result == int(result) else int(result) + 1
```

Comment:
Here you can use % and // for operations, as I understand you try to figure out if here is remainder from division